frontend/search/search_mobile.html

```diff
@@ -11,41 +11,57 @@
             </md-input-container>
         </form>
     </div>
-    <div class="search-mobile-options-title">
+    <div class="search-mobile-options-title" id="search-title">
```

Comment:
margin top and bottom should be the same

![image](https://user-images.githubusercontent.com/7011394/53579780-f3812a00-3b58-11e9-8f93-b948f8bb392d.png)